code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -78,12 +78,13 @@
 		for(var/mob/living/player in GLOB.mob_list)
 			if(player.client && player.stat != DEAD)
 				playercount += 1
-			if(player.client && player.mind && player.mind.special_role && player.stat != DEAD)
-				traitorcount += 1
-			if(player.client && player.mind && !player.mind.special_role && player.stat != DEAD)
-				if(ishuman(player) || isrobot(player) || isAI(player))
-					if(player.client && (ROLE_TRAITOR in player.client.prefs.be_special) && !jobban_isbanned(player, ROLE_TRAITOR) && !jobban_isbanned(player, "Syndicate"))
-						possible_traitors += player.mind
+				if(player.mind)
+					if(player.mind.special_role)
+						traitorcount += 1
+					else
+						if(ishuman(player) || isrobot(player) || isAI(player))
```

Comment:
You can put the else and if on the same line - Or use early return/continue as per our standard.